cmd/crc/cmd/config/config.go

```diff
@@ -19,7 +19,8 @@ var (
 	NameServer     = cfg.AddSetting("nameserver", nil, []cfg.ValidationFnType{cfg.ValidateIpAddress}, []cfg.SetFn{cfg.SuccessfullyApplied})
 	PullSecretFile = cfg.AddSetting("pull-secret-file", nil, []cfg.ValidationFnType{cfg.ValidatePath}, []cfg.SetFn{cfg.SuccessfullyApplied})
 
-	DisableUpdateCheck = cfg.AddSetting("disable-update-check", nil, []cfg.ValidationFnType{cfg.ValidateBool}, []cfg.SetFn{cfg.SuccessfullyApplied})
+	DisableUpdateCheck   = cfg.AddSetting("disable-update-check", nil, []cfg.ValidationFnType{cfg.ValidateBool}, []cfg.SetFn{cfg.SuccessfullyApplied})
+	ExperimentalFeatures = cfg.AddSetting("enable-experimental-features", nil, []cfg.ValidationFnType{cfg.ValidateBool}, []cfg.SetFn{cfg.SuccessfullyApplied})
```

Comment:
I could not check this on the mac, but I tried on linux and marked some parts of the preflights as experimental. You need the diff below for `--enable-experimental-features` (on the commandline) to work. `crc config enable-experimentatl-features` is still working fine after this patch.

```diff
diff --git a/cmd/crc/cmd/setup.go b/cmd/crc/cmd/setup.go
index d36725de..8fd61345 100644
--- a/cmd/crc/cmd/setup.go
+++ b/cmd/crc/cmd/setup.go
@@ -18,6 +18,7 @@ func init() {
 	setupCmd.Flags().StringVarP(&vmDriver, config.VMDriver.Name, "d",
 		machine.DefaultDriver.Driver, fmt.Sprintf("The driver to use for the OpenShift cluster. Possible values: %v", machine.SupportedDriverValues()))
 	setupCmd.Flags().Bool(config.ExperimentalFeatures.Name, false, "Allow the use of experimental features")
+	crcConfig.BindFlagSet(setupCmd.Flags())
 
 	rootCmd.AddCommand(setupCmd)
 }
```

Reply:
Not sure why this is needed... as the `vmdriver` worked like this (which should be removed actually; follow-up)

Reply:
I suspect the `vmdriver` parameter was buggy too, we probably did not use values set in the config.